server/services/unlockListService.ts

```diff
@@ -82,6 +83,18 @@ export default class UnlockListService {
       timeSlot,
     )
 
+    // popualte an array of prisoners with events in any searched activity
+    // if a prisoner has any category in the list the event should be added to the unlock items
+    const prisonersInAnyActivityCategory: string[] = []
```

Comment:
this is added due to the requirement to always show unselected categories if a prisoner is in any other selected category